.github/dependabot.yml

```diff
@@ -0,0 +1,72 @@
+version: 2
+
+updates:
+  # Update npm packages
+  - package-ecosystem: npm
+    directory: /
+    open-pull-requests-limit: 10
+
+    # Group packages into shared PR
+    groups:
+      build:
+        patterns:
+          - '@babel/*'
+          - 'babel-*'
+          - 'core-js'
+          - 'csso-cli'
+          - 'preact'
+          - 'source-map-loader'
+          - 'terser-*'
+          - 'webpack'
+          - 'webpack-*'
+
+      test:
+        patterns:
+          - '@wdio/*'
+          - 'chai'
+          - 'devtools'
+          - 'karma'
+          - 'karma-*'
+          - 'mocha'
+          - 'puppeteer'
+          - 'standard'
+          - 'webdriverio'
+
+      tools:
+        patterns:
+          - 'chalk'
+          - 'cross-env'
+          - 'dotenv'
+          - 'husky'
+          - 'npm-run-all'
+
+    reviewers:
+      - alphagov/design-system-developers
+
+    # Schedule run every Monday, local time
```

Comment:
Out of date comment – should now run on the [1st of every month](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#:~:text=Runs%20once%20each%20month.%20This%20is%20on%20the%20first%20day%20of%20the%20month.) (I guess that might be a weekend)